lib/fastlane/plugin/revenuecat_internal/actions/bump_phc_version_action.rb

```diff
@@ -106,6 +95,23 @@ def self.available_options
       def self.is_supported?(platform)
         true
       end
+
+      private_class_method def self.open_pr_against_main(automatic_release, github_pr_token, new_branch_name, new_version_number, repo_name, version_number)
+        Helper::RevenuecatInternalHelper.commit_changes_and_push_current_branch("Version bump for #{new_version_number}")
+
+        pr_title = "Updates purchases-hybrid-common to #{new_version_number}"
+        type_of_bump = Helper::VersioningHelper.detect_bump_type(version_number, new_version_number)
+        labels = ['dependencies']
+        labels << 'minor' if type_of_bump == :minor
```

Comment:
Hmm I was thinking about adding the `major` label for majors, but then again, we are not autobumping for majors I believe so not needed 👍